### PR TITLE
Add scikit-image example

### PR DESCRIPTION
**Pull request by tohmae**
This PR adds an scikit-image example that optimizes a classifier configuration for Olivetti faces dataset.
This PR addresses issue #557 .

### CODE REVIEW

**Comment by hvy** [comment]
Thanks @tohmae for addressing #557. The problem setting looks great. Could you just go through our [coding style guidelines](https://github.com/optuna/optuna/wiki/Coding-Style-Conventions) e.g. to not use camelCase function names and to keep your example as consistent as possible with the remaining code base?

Also, it'd be helpful if you could let us know what work remains to be done as it's titled WIP. 🙂

**Comment by tohmae** [comment]
Thank you for your comments. I confirmed [coding style guidelines](https://github.com/optuna/optuna/wiki/Coding-Style-Conventions) and reflected. Please review again.

**Comment by hvy** [comment]
Absolutely, I'll have another look. 

Could you just also add a link to this new example in `examples/README.md`[https://github.com/optuna/optuna/blob/master/examples/README.md] ?

**Comment by hvy** [comment]
Sorry for the delayed review. It basically LGTM. Would you mind taking a look at the minor comments above?

File: examples/skimage_lbp_simple.py [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    lbp = ft.local_binary_pattern(img, P, R, method)
+    if method == 'uniform':
+        hist, _ = np.histogram(lbp.ravel(), density=True,
+                               bins=np.arange(0, P + 3),
+                               range=(0, P + 2))
+    elif method == 'default':
+        hist, _ = np.histogram(lbp.ravel(), density=True,
+                               bins=np.arange(0, 2**P),
+                               range=(0, 2**P-1))
+    return hist
```
[the remark this answers is not in the document]
Comment (tohmae) [reply examples/skimage_lbp_simple.py]: OK.

File: examples/skimage_lbp_simple.py [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+def load_data():
+    rng = np.random.RandomState(0)
+    dataset = fetch_olivetti_faces(shuffle=True, random_state=rng)
+    faces = dataset.images
+    target = dataset.target
+    classes = sorted(list(set(target)))
+    ref_index = [(np.min(np.where(target == index))) for index in classes]
+    x_ref = faces[ref_index]
+    y_ref = target[ref_index]
+    test_index = [index for index in range(len(faces)) if index not in ref_index]
```
[the remark this answers is not in the document]
Comment (tohmae) [reply examples/skimage_lbp_simple.py]: Thank you. When program id modified, performance improve. 🙇

File: examples/skimage_lbp_simple.py [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+    x_test_hist = img2hist(x_test, P, R, method)
+    dist = calc_dist(x_ref_hist, x_test_hist, metric)
+
+    y_pred = np.argmin(dist, axis=1)
+    accuracy = sklearn.metrics.accuracy_score(y_test, y_pred)
+    return accuracy
+
+
+if __name__ == '__main__':
+    import optuna
```
[the remark this answers is not in the document]
Comment (tohmae) [reply examples/skimage_lbp_simple.py]: OK.

File: examples/skimage_lbp_simple.py [hunk 4]
```diff
@@ -0,0 +1,123 @@
+"""
+Optuna example that optimizes a configuration for Olivetti faces dataset using
+skimage.
+
+In this example, we optimize a classifier configuration for Olivetti faces dataset.
+We optimize paraterers of local_binary_pattern function in skimage and the
```
Comment (hvy) [inline examples/skimage_lbp_simple.py]: Seems like a typo.
```suggestion
We optimize parameters of local_binary_pattern function in skimage and the
```

**Comment by hvy** [comment]
Sorry an additional nitpick but there was a minor typo which I left as a comment. I'm also wondering if you could add this example to the [`examples/README.md`](https://github.com/optuna/optuna/blob/master/examples/README.md). If you're busy I could send a small patch afterwards so please let me know. Thanks.

**Comment by tohmae** [comment]
Thank you for your comments. I am sorry for typo.
I fixed as reviewed.

**Comment by tohmae** [comment]
Thank you for your comments. I am sorry for typo.
I fixed as reviewed.

**Pull request merged**